english/rickson-armbar/ricksonbyarmbar

```diff
@@ -0,0 +1,3 @@
+You just got armbarred. 

```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.